packages/mobile/ios/AudiusReactNative/Info.plist

```diff
@@ -20,7 +20,7 @@
     <!-- 
       Make sure this is above the currently released iOS version in the app store if your changes touch native code.
     -->
-		<string>1.1.61</string>
+		<string>1.1.63</string>
```

Comment:
```suggestion
		<string>1.1.62</string>
```
I think we didn't release 1.1.62 so you can use that. cc @sliptype

Reply:
Yeah 1.1.62 will be better!